.travis.yml

```diff
@@ -39,8 +39,9 @@ before_script:
 script:
   - bin/check-readme
   - bin/code-analysis
-  - bin/nosetests
   - bin/test
+  - bin/createcoverage
```

Comment:
Problem is that it then runs after `bin/test-plone_addon`, which leaves behind a `test.plone_addon` directory that the nosetests then also get run on, causing ImportErrors. Ah, but of course this directory only gets left behind when the command fails, so then there is no need to call createcoverage.  I will move it.

In that case, we should move createcoverage in the generated package too. I will move that too.